packages/backend/src/payment-method/ilp/service.ts

```diff
@@ -195,6 +196,10 @@ async function pay(
   try {
     const receipt = await Pay.pay({ plugin, destination, quote })
 
+    if (receipt.error) {
+      throw receipt.error
+    }
```

Comment:
move this check before the debug logger, so we don't log "'ILP payment completed'" when there is an error